### PR TITLE
Switch out the word 'resource' in some places.

### LINKED ISSUE #611: Update the command line help to avoid "resources" when possible
In a few places, the command line help says "resources" or "resource" when only one resource type is allowed, typically "container".

Overall, we should transition from saying "resources" to instead using "container" or "container|snapshot" if both are allowed (same goes for images).

### PR DESCRIPTION
**Pull request by hallyn**
Closes #611

Signed-off-by: Serge Hallyn <serge.hallyn@ubuntu.com>

### CODE REVIEW

**Comment by hallyn** [comment]
Thanks, going with that :)

**Comment by stgraber** [comment]
Need to update i18n :)

File: lxc/launch.go [hunk 1]
```diff
@@ -80,12 +80,12 @@ func (c *launchCmd) run(config *lxd.Config, args []string) error {
 
 	if name == "" {
 		if resp.Resources == nil {
-			return fmt.Errorf(gettext.Gettext("didn't get any affected resources from server"))
+			return fmt.Errorf(gettext.Gettext("didn't get any affected image, ccontainer or snapshot from server"))
```
Comment (chrisglass) [inline lxc/launch.go]: double c here

File: lxc/launch.go [hunk 2]
```diff
@@ -80,12 +80,12 @@ func (c *launchCmd) run(config *lxd.Config, args []string) error {
...
 		}
 
 		containers, ok := resp.Resources["containers"]
 		if !ok || len(containers) == 0 {
-			return fmt.Errorf(gettext.Gettext("didn't get any affected resources from server"))
+			return fmt.Errorf(gettext.Gettext("didn't get any affected image, ccontainer or snapshot from server"))
```
Comment (chrisglass) [inline lxc/launch.go]: Same

File: lxc/launch.go [hunk 3]
```diff
@@ -96,7 +96,7 @@ func (c *launchCmd) run(config *lxd.Config, args []string) error {
 		}
 
 		if count != 2 {
-			return fmt.Errorf(gettext.Gettext("bad number of things scanned from resource"))
+			return fmt.Errorf(gettext.Gettext("bad number of things scanned from image, ccontainer or snapshot"))
```
Comment (chrisglass) [inline lxc/launch.go]: And again (copy paste, I suspect :) )

Also, is "things" really what we want displayed?

File: po/lxd.pot [hunk 4]
```diff
@@ -510,7 +510,7 @@ msgid   "bad container url %s"
 msgstr  ""
 
 #: lxc/launch.go:99
-msgid   "bad number of things scanned from resource"
+msgid   "bad number of things scanned from image, ccontainer or snapshot"
```
Comment (chrisglass) [inline po/lxd.pot]: Same comment as above - but I suspect that's a "make i18n" away

File: po/lxd.pot [hunk 5]
```diff
@@ -543,7 +543,7 @@ msgid   "changing hostname of running containers not supported"
 msgstr  ""
 
 #: lxc/launch.go:83 lxc/launch.go:88
-msgid   "didn't get any affected resources from server"
+msgid   "didn't get any affected image, ccontainer or snapshot from server"
```
Comment (chrisglass) [inline po/lxd.pot]: I'm not sure adding yet another inline comment is useful, but here I go.

**Comment by stgraber** [comment]
Also needs rebasing.

File: lxc/launch.go (hunk 3 above)
Comment (hallyn) [inline lxc/launch.go]: eh, I didn't originate that so I leave that for someone else to fix :)  ("things")  it does sound clunky.

Comment (chrisglass) [inline lxc/launch.go]: Hehe fair enough :)

**Comment by stgraber** [comment]
Branch is a mess, looks like you did a merge instead of rebase :)

**Pull request merged**